README.md

```diff
@@ -18,7 +17,7 @@ indispensable help to this project.
 additional voice helper functions and features for DiscordGo.
 
 * See [dca](https://github.com/bwmarrin/dca) for an **experimental** stand alone
-tool that wraps `ffmpeg` to create opus encoded audio appropriate for use with
+tool that wraps `FFmpeg to create opus encoded audio appropriately for use with
```

Comment:
It seems like you dropped the closing code tick here.

Reply:
Good catch.